app/services/activitypub/fetch_remote_account_service.rb

```diff
@@ -18,38 +20,50 @@ def call(uri, id: true, prefetched_body: nil, break_on_redirect: false, only_key
       else
         body_to_json(prefetched_body, compare_id: id ? uri : nil)
       end
+    rescue Oj::ParseError
+      raise ParsingError
```

Comment:
I don't see where this is defined... Or what's the point of re-defining Oj::ParseError

Reply:
It indeed wasn't defined, leftover from a previous version of the patch.

Previously, `Oj::ParseError` were silently ignored, I figure it makes sense to consider it as an “expected failure mode” in that service and return an appropriate error, without the caller having to know about each expected failure modes specifically.